README.md

```diff
@@ -33,6 +33,15 @@ Here is an overview of the architecture of GUAC:
 
 ![image](https://user-images.githubusercontent.com/3060102/182689908-477f4770-1142-4c18-8fa9-16d93dcf84b4.png)
 
+## Supported input formats
+
+- [CycloneDX](https://github.com/CycloneDX/specification)
+- [Dead Simple Signing Envelope](https://github.com/secure-systems-lab/dsse)
+- [OSV](https://ossf.github.io/osv-schema/)
```

Comment:
We. support OSV by proxy of certification attestations. So maybe minus this and then add [Intoto ITE6](https://github.com/in-toto/attestation) as another input format.. @pxp928  can confirm

Reply:
Thank you. Adjusted based on your comments. PTAL.